router/batchrouter/asyncdestinationmanager/snowpipestreaming/types.go

```diff
@@ -0,0 +1,144 @@
+package snowpipestreaming
+
+import (
+	"context"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/mitchellh/mapstructure"
+
+	"github.com/rudderlabs/rudder-go-kit/config"
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	"github.com/rudderlabs/rudder-go-kit/stats"
+
+	"github.com/rudderlabs/rudder-server/router/batchrouter/asyncdestinationmanager/snowpipestreaming/internal/model"
+
+	backendconfig "github.com/rudderlabs/rudder-server/backend-config"
+	whutils "github.com/rudderlabs/rudder-server/warehouse/utils"
+)
+
+type (
+	Manager struct {
+		conf         *config.Config
```

Comment:
conf and config keys are confusing rename one of them

Reply:
Renamed conf to appConfig.